### DISCUSSION TITLE
Users option

### DISCUSSION THREAD

**Discussion by myuyu** [hahwul/gitls#3] [Ideas]
Hey i think adding an option to extract users URL Profile and then pass them to the main function to grab repos urls from them will be great option
Command : `echo "https://github.com/paypal" | gitls`
Result : grab paypal repos urls + grab repos urls from paypal users accounts

**Comment by myuyu** [comment]
it can be done through `curl -H "Authorization: token $t" https://api.github.com/orgs/paypal/members | jq ".[] | .html_url" -r | gitls |xargs -I % gitleaks --repo-url=% -v` but if its an option will be great then this

**Reply by hahwul** [reply]
Thank you for your opinion. I'll add the flag names below and develop it.

E.g
-github-access-token
-include-accounts

And github access token will also support input through env😄

**Reply by myuyu** [reply]
Thank you so much this will be really helpful

**Reply by hahwul** [reply]
Hello again @myuyu 
Added the `-include-account` option. Available by updating

## Testing
run 
```
echo https://github.com/paypal | ./gitls -include-account
```

result 
```
....
https://github.com/paypal/tech-talks
https://github.com/paypal/TLS-update
https://github.com/paypal/yurita
https://github.com/ahunnargikar
https://github.com/ahunnargikar/docker-chronos-image
https://github.com/ahunnargikar/docker-tomcat7
https://github.com/ahunnargikar/DockerConDemo
https://github.com/ahunnargikar/elasticsearch-registry-backend
https://github.com/ahunnargikar/elasticsearchindex
https://github.com/ahunnargikar/jenkins-dind
https://github.com/ahunnargikar/jenkins-standalone
https://github.com/ahunnargikar/vagrant-mesos
https://github.com/ahunnargikar/vagrant_docker_registry
https://github.com/anandpalanisamy
https://github.com/anilgursel
https://github.com/anilgursel/squbs-sample
https://github.com/bluepnume
https://github.com/bluepnume/deprecate.js
```

**Reply by hahwul** [reply]
## Updating 
from source 
```
GO111MODULE=on go get -v github.com/hahwul/gitls
```

homebrew
```
brew upgrade gitls
```

snapcraft
```
$ sudo snap refresh gitls
```

**Comment by myuyu** [comment]
Ohh Nice ,  thank you so much <3

**Reply by hahwul** [reply]
@myuyu 
and I named the flag randomly, but if you have a better name, please share it with me! I'll add it to the next version :D

**Reply by myuyu** [reply]
I think `-include-users` is clear

**Reply by hahwul** [reply]
@myuyu 
thank you :D
It's a much better name. I fixed it right away.
```
$ ./gitls -v
v1.0.2

$ ./gitls -h
Usage of ./gitls:
  -include-users
    	include repo of org users(member)
  -l string
    	List of targets (e.g -l sample.lst)
  -o string
    	write output file (optional)
  -proxy string
    	using custom proxy
  -tor
    	using tor proxy / localhost:9050
  -version
    	version of gitls

```
